src/backend/downloadmanager/downloadqueue.ts

```diff
@@ -179,6 +179,12 @@ function getQueueInformation() {
 
 function cancelCurrentDownload({ removeDownloaded = false }) {
   if (currentElement) {
+    if (currentElement.params.installDlcs) {
+      const dlcsToRemove = currentElement.params.installDlcs as string[]
```

Comment:
Not sure if we can currently get here with GOG games, but
```suggestion
    if (Array.isArray(currentElement.params.installDlcs)) {
      const dlcsToRemove = currentElement.params.installDlcs
```
might be safer